modules/custom/az_news/az_news_export/config/install/views.view.az_news_export.yml

```diff
@@ -0,0 +1,1635 @@
+        - 'config:field.storage.node.field_az_news_tags'
+        - 'config:field.storage.node.field_az_published'
+        - 'config:field.storage.node.field_az_short_title'
+        - 'config:field.storage.node.field_az_subheading'
+        - 'config:field.storage.node.field_az_summary'
+  rest_export_1:
+    id: rest_export_1
+    display_title: 'REST export'
+    display_plugin: rest_export
+    position: 2
+    display_options:
+      pager:
+        type: mini
+        options:
+          offset: 0
+          items_per_page: 25
+          total_pages: null
+          id: 0
+          tags:
+            next: ››
+            previous: ‹‹
+          expose:
+            items_per_page: false
+            items_per_page_label: 'Items per page'
+            items_per_page_options: '5, 10, 25, 50'
+            items_per_page_options_all: false
+            items_per_page_options_all_label: '- All -'
+            offset: false
+            offset_label: Offset
+      style:
+        type: serializer
+        options:
+          formats:
+            json: json
+      row:
+        type: az_news_data_field
+        options:
+          field_options:
+            title:
+              alias: ''
+              raw_output: '1'
+            nid:
+              alias: ''
+              raw_output: 0
+            view_node:
+              alias: canonical_url
+              raw_output: 0
+            uid:
+              alias: author
+              raw_output: 0
+            created:
+              alias: ''
+              raw_output: 0
+            changed:
+              alias: ''
+              raw_output: 0
+            field_az_body:
+              alias: ''
+              raw_output: 0
+            field_az_summary:
+              alias: ''
+              raw_output: 0
+            field_az_summary_1:
+              alias: field_az_summary_plain
+              raw_output: 0
+            field_az_short_title:
+              alias: ''
+              raw_output: '1'
+            field_az_subheading:
+              alias: ''
+              raw_output: 0
+            field_az_byline:
+              alias: ''
+              raw_output: 0
+            field_az_published:
+              alias: ''
+              raw_output: 0
+            field_az_expiration_date:
+              alias: ''
+              raw_output: 0
+            field_az_news_tags:
+              alias: ''
+              raw_output: '1'
+            field_az_enterprise_attributes:
+              alias: ''
+              raw_output: '1'
+            field_az_attachments:
+              alias: ''
+              raw_output: '1'
+            field_az_contacts:
+              alias: ''
+              raw_output: '1'
+            field_az_media_image:
+              alias: ''
+              raw_output: '1'
+            field_az_media_thumbnail_image:
+              alias: ''
+              raw_output: '1'
+      display_extenders: {  }
+      path: az_quickstart/1.0/export/az_news
```

Comment:
We need to decide for sure if this is the URL pattern we want to use for this API.